packages/flutter_driver/lib/src/driver/web_driver.dart

```diff
@@ -248,3 +227,30 @@ class FlutterWebConnection {
     _driver.quit();
   }
 }
+
+/// Utility functions for working with WebDriver.
+class DriverUtils {
```

Comment:
I don't think there isn't any reason to make a class with only static members in Dart. Prefer creating top-level functions (see https://dart.dev/guides/language/effective-dart/design#avoid-defining-a-class-that-contains-only-static-members )

Reply:
Done.